### PR TITLE
source-height contribution from btkcodedev

### PR DESCRIPTION
**Pull request by btkcodedev**
Closes https://github.com/airbytehq/airbyte-internal-issues/issues/9675
Auto-generated PR by the Connector Builder for source-height

### CODE REVIEW

File: docs/integrations/sources/height.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Website: https://height.app
+
+API Documentation: https://height.notion.site/API-documentation-643aea5bf01742de9232e5971cb4afda
+
+## Configuration
+
+| Input | Type | Description | Default Value |
+|-------|------|-------------|---------------|
+| `api_key` | `string` | API secret which is copied from the settings page of height.app  |  |
+| `start_date` | `string` | Start date for   |  |
```
Comment (natikgadzhi) [inline docs/integrations/sources/height.md:17]: @btkcodedev add the start date docs here, this sentence seems unfinished.

**Review by natikgadzhi — APPROVED** [review]
@btkcodedev just one doc change that I want to see, otherwise awesome!

File: docs/integrations/sources/height.md (hunk 1 above)
Comment (btkcodedev) [reply docs/integrations/sources/height.md]: Done

**Pull request merged**